scaleway/resource_storage_object_bucket_test.go

```diff
@@ -0,0 +1,100 @@
+package scaleway
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("scaleway_storage_object_bucket", &resource.Sweeper{
+		Name: "scaleway_storage_object_bucket",
+		F:    testSweepStorageObjectBucket,
+	})
+}
+
+func testSweepStorageObjectBucket(region string) error {
+	s3client, err := sharedS3ClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %s", err)
+	}
+
+	listBucketResponse, err := s3client.ListBuckets(&s3.ListBucketsInput{})
+	if err != nil {
+		return fmt.Errorf("couldn't list buckets: %s", err)
+	}
+
+	for _, bucket := range listBucketResponse.Buckets {
+		log.Println(*bucket.Name)
```

Comment:
Please use [this logger](https://github.com/terraform-providers/terraform-provider-scaleway/blob/master/scaleway/logger.go#L17).

Reply:
What do you think about:
```go
log.Printf("Deleting %s bucket", *bucket.Name)
```
?

Reply:
👍